include/fmt/std.h

```diff
@@ -389,6 +389,29 @@ struct formatter<
 #endif
   }
 };
-FMT_END_NAMESPACE
 
+FMT_EXPORT
+template <typename T, typename Char>
+struct formatter<std::atomic<T>, Char> : formatter<T, Char> {
+  template <typename FormatContext>
+  FMT_CONSTEXPR20 auto format(const std::atomic<T>& v, FormatContext& ctx) const
```

Comment:
``std::atomic<T>::load`` is not constexpr:
https://en.cppreference.com/w/cpp/atomic/atomic/load

Reply:
fixed